_includes/linkers/issues.md

```diff
@@ -2,7 +2,8 @@
 <!--
 issues.md: creates Markdown referency-style links to issues and pull
 requests in the Bitcoin Core, LND, C-Lightning, libsecp256k1, BIPs, BOLTs,
-and Rust-Lightning repositories.
+and Rust-Lightning repositories, and Bitcoin Core PR Review Club archive
+pages.
```

Comment:
Nit: two find "ands" here.  At this point, we can probably just remove the list of repositories and let anyone interested read the code.  With each repository on a separate line, I think it's easy enough to figure out.